src/any_agent/config.py

```diff
@@ -122,6 +123,8 @@ class TracingConfig(BaseModel):
     cost_info: bool = True
     """Whether traces should include cost information"""
 
+    additional_exporters: list[SpanExporter] = Field(exclude=True, default=[], repr=False)
+
```

Comment:
This is no longer being added as a part of this PR, right?

Reply:
Hum, no, sorry. These should be removed.